t/run-complex-model-with-filters.t

```diff
@@ -59,6 +59,29 @@ for <hr-inclNOxExtended hr-inclNOxExtendedWithFilters> -> $model-version {
         is (+%output-hash<Storage><tan_into_application>).round(.001), $tan-into-application.round(.001),
                 "Correct tan_into_application result: { (+%output-hash<Storage><tan_into_application>).round(.001) }";
 
+        # check balance for each "Stufe" (livestock, storage, application)
+        # input = output + loss + remaining  
+        if ($model-version eq "hr-inclNOxExtendedWithFilters") { 
+            ### check livestock ntot balance
+            my $balance-livestock-ntot = %output-hash<Livestock><n_excretion>;
+            my @subtract =
```

Comment:
I would suggest a variable name that fits the elements of the array. `subtract` describes what you are doing with it, not what's in there. Also, I would use plural for arrays and in the for loop would assign to the singular of the same word. I guess Jonathan's example with `@inputs`and `$input`  doesn't fit, because that's not what those are. May `part/s`, `output/s`, `loss/es`, ...